RijsMuseum/app/src/main/java/com/example/rijsmuseum/MainActivity.kt

```diff
@@ -3,64 +3,24 @@ package com.example.rijsmuseum
 import android.os.Bundle
 import android.view.View
 import androidx.appcompat.app.AppCompatActivity
-import androidx.lifecycle.ViewModelProvider
-import androidx.recyclerview.widget.LinearLayoutManager
-import com.example.network.getters.CollectionsGetter
-import com.example.network.models.Collections
-import com.example.rijsmuseum.adapter.RecyclerAdapter
 import com.example.rijsmuseum.databinding.ActivityMainBinding
-import com.example.rijsmuseum.fragments.DetailsFragment
-import com.example.rijsmuseum.viewmodel.MainViewModel
+import com.example.rijsmuseum.fragments.ArtifactFragment
 
-class MainActivity : AppCompatActivity(), RecyclerAdapter.OnItemClickListener {
-    private lateinit var viewModel: MainViewModel
+class MainActivity : AppCompatActivity() {
     private lateinit var binding: ActivityMainBinding
 
-    private lateinit var adapter: RecyclerAdapter
-
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         binding = ActivityMainBinding.inflate(layoutInflater)
         setContentView(binding.root)
 
-        viewModel = ViewModelProvider(this).get(MainViewModel::class.java)
-
-        adapterSettings()
-
         //Button to get the list of artifacts
         binding.getButton.setOnClickListener {
```

Comment:
what do you still need the button for? When you go to Facebook/Instagram - are you presented with a button to click before showing you the feed. Or is it just shown when you open the app?